config/settings.yml

```diff
@@ -995,6 +995,15 @@ genisis:
   user: TestUser
   pass: bogus
 
+# Settings for connecting AFS Veteran Services
+dgi:
```

Comment:
Did you add these setting to the various dev, staging, prod, sandbox files in the devops repo? I didn't see any links in the PR description

Reply:
This is the PR that added those to staging, and sandbox. [DevOps PR for staging and sandbox env](https://github.com/department-of-veterans-affairs/devops/pull/10152). There is a pending PR for Prod [DevOps PR for Prod is pending](https://github.com/department-of-veterans-affairs/devops/pull/10153)

Reply:
Thanks!